lib/zypper.pm

```diff
@@ -41,7 +41,7 @@ lock so that we can run a new zypper for our test.
 
 =cut
 sub wait_quit_zypper {
-    assert_script_run 'until ! pgrep zypper; do sleep 1; done';
+    assert_script_run('until ! pgrep \'zypper|purge-kernels|rpm\' > /dev/null; do sleep 10; done', 600);
```

Comment:
@lemon-suse I've added you as a reviewer for this change

Reply:
It looks better and robust than before.

Reply:
Just have a question, you have found such situation that not a zypper process but need grep by 'purge-kernels' or 'rpm'? Just feel interest about it.

Reply:
yep, you can find the reason in the PR description (point 3).
`purges-kernel` doesn't use `zypper` command on 15-SP1 but only `rpm -e`

Reply:
That's make sense. :)